_data/guide.yml

```diff
@@ -61,6 +61,11 @@ docs:
   description: 
   icon: <i class="far fa-file-alt fa-5x" aria-hidden="true"></i>
 
+- title: Semantic HTML
+  url: /semantic-html
+  description:
+  icon: <i class="far fa-code fa-5x" aria-hidden="true"></i>
```

Comment:
@jaube05 fa-code for some reason didn't generate an icon https://accessibility-qa.civicactions.com/guide/, maybe choose a different one?

Reply:
Done